src/ReportManager/DetailsSection/index.js

```diff
@@ -67,7 +72,7 @@ const DetailsSection = ({
 }, ref) => {
   const dispatch = useDispatch();
   const { t } = useTranslation('reports', { keyPrefix: 'reportManager.detailsSection' });
-  const isNewFormSchema = false;
+  const isNewFormSchemaSupportEnabled = DEVELOPMENT_FEATURE_FLAGS[FEATURE_FLAG_LABELS.EFB_FORM_SCHEMA_SUPPORT_ENABLED];
```

Comment:
This should be a constant at the top of the file. We don't need to define a property in the scope of the component really 😄 
```
const EFB_FORM_SCHEMA_SUPPORT_ENABLED = DEVELOPMENT_FEATURE_FLAGS[FEATURE_FLAG_LABELS.EFB_FORM_SCHEMA_SUPPORT_ENABLED];
```

Reply:
Got it 🫡